src/chi/components/brand/brand.scss

```diff
@@ -0,0 +1,22 @@
+@import 'mixins';
+@import 'variables';
+
+$brand-sizes: (xs: 0.75rem, sm: 1rem, sm2: 1.25rem, sm3: 1.5rem, md: 2rem, lg: 4rem, xl: 6rem, xxl: 8rem);
+
+.chi {
+  .a-brand {
+    & > img {
```

Comment:
@davidojedacl add > svg so it render properly

Reply:
Yes, would be great to support both `img` and `svg` tags within a-brand